src/azure-cli/azure/cli/command_modules/vm/azure_stack/_vm_utils.py

```diff
@@ -118,18 +118,18 @@ def get_key_vault_base_url(cli_ctx, vault_name):
 
 
 def list_sku_info(cli_ctx, location=None):
-    from ._client_factory import _compute_client_factory
+    from ..aaz.latest.vm import ListSkus as _ListSkus
```

Comment:
May I ask why to call the latest profile of ListSkus in the old profiles?

Reply:
Using import aaz by profile